tasks/task.json

```diff
@@ -43,6 +43,33 @@
             }
           }
         }
+      },{ 
+        "type": "object",
```

Comment:
Again this is wrong, `files` should be added as an optional property at line 35.